13-TeV-examples/uproot_python/HyyAnalysisNew.ipynb

```diff
@@ -11,93 +11,107 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# How to rediscover the Higgs boson yourself!\n",
+    "## How to rediscover the Higgs boson yourself!\n",
```

Comment:
Is there a reason why this is not a heading level 1 (meaning: just one #)?

Reply:
Should be alright now :)